### DISCUSSION TITLE
Short course as first cookbook in Spanish

### DISCUSSION THREAD

**Discussion by aladinor** [ProjectPythia/projectpythia.github.io#372] [General]
Hi everyone, 

Following up on what we discussed yesterday during the Pythia meeting I would like to share the GitHub link from AtmosCol2023

https://github.com/aladinor/Atmoscol2023

Please let me know your thoughts and comments. 

Cheers, 

Alfonso

**Comment by brian-rose** [comment]
Hi @aladinor, there was a pretty strong consensus at the Community Meeting that we should move forward with housing this book in the Pythia Cookbook gallery.

Following steps in the [Cookbook contributors guide](https://github.com/ProjectPythia/.github/blob/main/.github/CONTRIBUTING.md#transfer-cookbook-to-the-projectpythia-organization), I think the next action should be to transfer ownership of the repository to the ProjectPythia organization.

I went ahead and invited you as a member of the organization.

**Reply by aladinor** [reply]
Hi Brian, 

I'm sorry for not getting back to you sooner. I already transferred ownership to the ProjectPythia organization. Please let me know if something else is required.
